### ISSUE TITLE
shift-right-click-repairs-mending-points don't working

### ISSUE THREAD

**Issue by diznover** [pl3xgaming/Purpur#1268]
### Spark link

it does not matter

### Expected behavior

When I right-click with a tool tool enhanced by mending in the main hand, the tool will repaired

### Observed/Actual behavior

nothing happens

### Steps/models to reproduce

change 0 to 1 or higher number in shift-right-click-repairs-mending-points in purpur.yml
start the server
login to server
try to repair tool with mending doing shift right-click with with this tool in main hand

I hope I described it according to the method [ELI5](https://www.dictionary.com/e/slang/eli5/)

### Purpur version

Current: git-Purpur-1913 (MC: 1.19.3)*
Previous: git-Purpur-1912 (MC: 1.19.3)
* You are running the latest version

### Agreements

- [X] I am running the latest version of Purpur available from https://purpurmc.org/downloads.
- [X] I have searched for and ensured there isn't already an open issue regarding this.
- [ ] I ticked all the boxes without actually reading them
- [X] My version of Minecraft is supported by Purpur.

**Comment by RhythmicSys** [comment]
Has the player been given the permission `purpur.mending_shift_click` ?

**Comment by diznover** [comment]
No, now I set the permission and now everything works, thank you very much!

**Comment by RhythmicSys** [comment]
awesome!
